packages/beacon-node/src/sync/unknownBlock.ts

```diff
@@ -437,6 +464,58 @@ export class UnknownBlockSync {
     }
   }
 
+  private async fetchUnavailableBlock(
```

Comment:
the name is confusing here as we have block but miss some blobs, maybe `fetchUnavailableBlobs`? also drop some comments on method header